### PR TITLE
Added numerous things

### PR DESCRIPTION
**Pull request by Aryan-Naveen**
1. The method of the rotated rectangle for the vertices returned points that make sense
2. Rewrote test and placed them inside a learning folder
3. Returns an array of Point

### CODE REVIEW

**Comment by mandrews281** [comment]
One overall comment.  We need to decide whether to keep the team281 level in the 2019 code.  The WPI sample projects have all removed it and it is not in the Robot2019 project.  I realize that this code will run on the RPi and not (necessarily) the RoboRio; but if some useful code gets developed here that we DO want to move to the main robot, then having the extra level in the code will hamper the merge effort.  I ran into this with the port of 2018 Macarena to 2019.

**Review by mandrews281 — COMMENTED** [review]
Another comment.  There are some binary files from the .gradle directory which should probably not be in the repo.  Can we remove them?

File: src/main/java/frc/team281/robot/ConvertRectToPoint.java [hunk 1]
```diff
@@ -1,121 +1,128 @@
...
+        vertices.add(topLeft);
+
+        return vertices;
+    }
+
+    public static ArrayList<Point> rotateAroundOrigin(ArrayList<Point> vertices, RotatedRect rect) {
+        
+        for(Point p: vertices){
+            DecimalFormat df = new DecimalFormat("#, ###.##");
+            String xPrime = df.format(p.x * Math.cos(Math.PI-rect.angle) - p.y * Math.sin(Math.PI-rect.angle));
```
Comment (mandrews281) [inline src/main/java/frc/team281/robot/ConvertRectToPoint.java]: I think you have problems here.  RotatedRect.angle is measured in degrees while Math.cos etc want angles in radians.  You need to take your rect.angle and multiple by (Math.PI/180.0)

Comment (mandrews281) [reply src/main/java/frc/team281/robot/ConvertRectToPoint.java]: From: https://docs.opencv.org/3.4/db/dd6/classcv_1_1RotatedRect.html
"Each rectangle is specified by the center point (mass center), length of each side (represented by Size2f structure) and the rotation angle in degrees."

File: src/test/java/frc/team281/test/learning/TestConvertRectToPoint.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+
+public class TestConvertRectToPoint {
+    @Test
+    public void TestConvertRectToPoint2(){
+
+        
+        Point center1 = new Point(4,5);
+        Size s1 = new Size(4, 8);
+        double a1 = Math.PI/4;
```
Comment (mandrews281) [inline src/test/java/frc/team281/test/learning/TestConvertRectToPoint.java]: Don't put radians in the angle of a RotatedRect.  OpenCV will NOT like that.

**Comment by Aryan-Naveen** [comment]
Fixed those few issues

**Pull request merged**